controllers/ironicinspector_controller.go

```diff
@@ -932,10 +932,10 @@ func (r *IronicInspectorReconciler) generateServiceConfigMaps(
 			Labels:        cmLabels,
 		},
 	}
-	err = configmap.EnsureConfigMaps(ctx, h, instance, cms, envVars)
+	err := configmap.EnsureConfigMaps(ctx, h, instance, cms, envVars)
 	if err != nil {
 		r.Log.Error(err, "Unable to create Config Maps %v")
```

Comment:
Should we align this with the other controllers?
i.e not log - just do the
 `return configmap.EnsureConfigMaps(ctx, h, instance, cms, envVars)`

Reply:
Yeah let's make it consistent. I hesitated to remove this because the log might be useful but at the same time I didn't know it's worth updating all operators to put this error (This is the only logic I can find in service operators to leave error record).

Reply:
I added the log there for debugging. Since nothing would surface without it.
But with this change not ignoring error I expect the caller would log/handle the error.